commit 93860685d1877f20b4266917f088ec09b7396ec1
Author: Chris Burroughs <chris.burroughs@joyent.com>
Date:   2019-06-14T21:29:37-04:00 (4 months ago)
    
    TRITON-1724 sapi should use node v6.x
    TRITON-1725 update sapi base image to triton-origin-x86_64-18.4.0
    Reviewed by: Trent Mick <trentm@gmail.com>
    Reviewed by: Pedro Palazón Candel <pedro@joyent.com>
    Approved by: Trent Mick <trentm@gmail.com>
    Approved by: Pedro Palazón Candel <pedro@joyent.com>

diff --git a/Makefile b/Makefile
index 32283e0..f82f742 100644
--- a/Makefile
+++ b/Makefile
@@ -32,11 +32,10 @@ JSSTYLE_FILES	 = $(JS_FILES)
 JSSTYLE_FLAGS    = -o indent=4,doxygen,unparenthesized-return=0
 SMF_MANIFESTS	 = smf/manifests/sapi.xml
 
-NODE_PREBUILT_VERSION=v4.9.0
+NODE_PREBUILT_VERSION=v6.17.0
 ifeq ($(shell uname -s),SunOS)
-	NODE_PREBUILT_TAG=zone
-	# The sdcnode matching image for triton-origin-multiarch-15.4.1 images.
-	NODE_PREBUILT_IMAGE=18b094b0-eb01-11e5-80c1-175dac7ddf02
+	NODE_PREBUILT_TAG=zone64
+	NODE_PREBUILT_IMAGE=c2c31b00-1d60-11e9-9a77-ff9f06554b0f
 endif
 
 ENGBLD_USE_BUILDIMAGE	= true
@@ -73,7 +72,8 @@ SVC_TARBALL 	:= $(NAME)-pkg-$(STAMP).tar.gz
 SVC_PKGDIR	:= $(TOP)/$(BUILD)/service
 SVC_INSTDIR	:= $(SVC_PKGDIR)/root/opt/smartdc/sapi
 
-BASE_IMAGE_UUID = 04a48d7d-6bb5-4e83-8c3b-e60a99e0f48f
+# our base image is triton-origin-x86_64-18.4.0
+BASE_IMAGE_UUID = a9368831-958e-432d-a031-f8ce6768d190
 BUILDIMAGE_NAME = $(NAME)
 BUILDIMAGE_DESC	= SDC SAPI
 BUILDIMAGE_PKG	= $(PWD)/$(SVC_TARBALL)
diff --git a/deps/eng b/deps/eng
index d25b8fc..ca4a08e 160000
--- a/deps/eng
+++ b/deps/eng
@@ -1 +1 @@
-Subproject commit d25b8fc60fb8c649559924870fe3aaf75e7421d5
+Subproject commit ca4a08e8b7bba7a86d897060338fba27ee4e872b
